code/src/qik_2s12v10.cpp

```diff
@@ -3,25 +3,12 @@
 namespace r2d2::moving_platform {
 
 
-    qik_2s12v10_c::qik_2s12v10_c(r2d2::uart_ports_c &uart_port,
-                                 unsigned int baud_rate,
+    qik_2s12v10_c::qik_2s12v10_c(r2d2::usart::usart_ports &usart_port, unsigned int baud_rate,
                                  hwlib::pin_out &reset_pin): 
-        reset_pin(reset_pin), 
-        usart_bus(baud_rate, uart_port) 
+        reset_pin(reset_pin),
+        usart_bus(baud_rate, usart_port) 
```

Comment:
the `{}` on the next line should be on this line

Reply:
Your commit has fixed this